docusaurus/docs/reactnative/guides/message_customizations.mdx

```diff
@@ -507,3 +507,26 @@ The `deleteMessagesVisibiltyType` prop in the Channel component enables changing
   deletedMessagesVisibiltyType="sender"
 >
 ```
+
+## Memoization handler for custom message properties
+
```

Comment:
Awesome, let's Stream's users our called customers and maybe we should emphasize that this is not an issue by replacing the word `this should be fixed` with `this can be achieved`.

Same goes for ton of voice from the customer perspective, instead of `when we add` something like `when a custom property is added...`